src/TBuffer.h

```diff
@@ -174,7 +181,7 @@ class TBuffer
     void translateToPlainText(std::string& s, bool isFromServer=false);
     void append(const QString& chunk, int sub_start, int sub_end, const QColor& fg, const QColor& bg, const TChar::AttributeFlags flags = TChar::None, const int linkID = 0);
     // Only the bits within TChar::TestMask are considered for formatting:
-    void append(const QString& chunk, const int sub_start, const int sub_end, const TChar format, const int linkID = 0);
+    void append(const QString &chunk, const int sub_start, const int sub_end, const TChar format, const int linkID = 0);
```

Comment:
Not in accordance with our Clang-format settings I think...!

Reply:
Fixed.